Mathlib/Algebra/BigOperators/Group/Finset/Basic.lean

```diff
@@ -609,81 +599,39 @@ This is a multiplicative discrete analogue of the fundamental theorem of calculu
 verify that it's equal to a different function just by checking differences of adjacent terms.
 
 This is a discrete analogue of the fundamental theorem of calculus."]
-theorem prod_range_induction (f s : ℕ → β) (base : s 0 = 1)
+theorem prod_range_induction (f s : ℕ → M) (base : s 0 = 1)
     (step : ∀ n, s (n + 1) = s n * f n) (n : ℕ) :
     ∏ k ∈ Finset.range n, f k = s n := by
   induction n with
   | zero => rw [Finset.prod_range_zero, base]
   | succ k hk => simp only [hk, Finset.prod_range_succ, step, mul_comm]
 
-/-- A telescoping product along `{0, ..., n - 1}` of a commutative group valued function reduces to
-the ratio of the last and first factors. -/
-@[to_additive "A telescoping sum along `{0, ..., n - 1}` of an additive commutative group valued
-function reduces to the difference of the last and first terms."]
-theorem prod_range_div {M : Type*} [CommGroup M] (f : ℕ → M) (n : ℕ) :
-    (∏ i ∈ range n, f (i + 1) / f i) = f n / f 0 := by apply prod_range_induction <;> simp
```

Comment:
Was this lemma, and the others in this part of the diff, moved unintentionally? Possibly as the result of a bad merge? Please fix if so. If intentional, please explain in the PR description.

Reply:
These are intentional and already mentioned in the PR description:
> Also order according to typeclass assumptions